### PR TITLE
Add HTCondor-CE Intro video

### PR DESCRIPTION
**Pull request by brianhlin**
And update phrasing on the homepage to reflect language in the HTCondor-CE Translational CS paper

### CODE REVIEW

**Comment by brianhlin** [comment]
Another minor commit for the overview doc snuck in. I've got more changes coming for that doc but figured it should go into a separate PR because of its scope

File: docs/index.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
-Supported batch systems include:
+For an introduction to HTCondor-CE, watch our [recorded webinar](https://www.youtube.com/embed/6IWaMbofG7M) from the EGI
+Community Webinar Programme:
 
-- [Grid Engine](http://www.univa.com/products/)
-- [HTCondor](http://htcondor.org)
-- [LSF](https://www.ibm.com/us-en/marketplace/hpc-workload-management)
-- [PBS Pro](https://www.pbsworks.com/PBSProduct.aspx?n=Altair-PBS-Professional&c=Overview-and-Capabilities)/[Torque](http://www.adaptivecomputing.com/products/torque/)
-- [Slurm](https://slurm.schedmd.com/)
+<iframe width="560" height="315" src="https://www.youtube.com/embed/6IWaMbofG7M" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
```
Comment (matyasselmeci) [inline docs/index.md:19]:
```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/6IWaMbofG7M" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
```
please? (Assuming that does what I think it does, i.e. turn off autoplay.)

Comment (brianhlin) [reply docs/index.md:19]: Looking at their docs, I suspect that just allows us to set autoplay if we add `&autoplay=1` to the URL: https://support.google.com/youtube/answer/171780?hl=en

**Comment by brianhlin** [comment]
Strangely GitHub has two Travis CI checks that refer to the same builds but it's stuck waiting on one, even though the builds are done (https://github.com/htcondor/htcondor-ce/pull/347/checks?check_run_id=785811214). Using admin privs to merge.

@JoshKarpel you know a thing or two about Travis, right? Any idea what that may be about?

**Comment by JoshKarpel** [comment]
:man_shrugging:  looks like a GitHub integration bug. The problems I used to have were related to the `.org -> .com` switchover, which I don't think is the issue here.

**Pull request merged**